packages/story-editor/src/components/panels/design/animation/animation.js

```diff
@@ -101,6 +106,11 @@ function AnimationPanel({
       .filter((a) => !a.delete);
   }, [selectedElements, selectedElementAnimations]);
 
+  const isFirstPage = useMemo(
+    () => currentPageNumber === 1,
+    [currentPageNumber]
+  );
```

Comment:
I think we can remove this and just accomplish the same thing with the selector. This approach will also cause less empty re-renders